userapi/src/test/java/fk/prof/userapi/api/MockAggregatedProfile.java

```diff
@@ -0,0 +1,419 @@
+package fk.prof.userapi.api;
+
+import com.amazonaws.ClientConfiguration;
+import com.amazonaws.Protocol;
+import com.amazonaws.auth.AWSCredentials;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.services.s3.AmazonS3Client;
+import com.amazonaws.services.s3.S3ClientOptions;
+import com.amazonaws.services.s3.model.*;
+import com.codahale.metrics.MetricRegistry;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.protobuf.CodedOutputStream;
+import fk.prof.aggregation.AggregatedProfileNamingStrategy;
+import fk.prof.aggregation.model.*;
+import fk.prof.aggregation.proto.AggregatedProfileModel;
+import fk.prof.aggregation.state.AggregationState;
+import fk.prof.storage.AsyncStorage;
+import fk.prof.storage.ObjectNotFoundException;
+import fk.prof.storage.buffer.ByteBufferPoolFactory;
+import fk.prof.userapi.Deserializer;
+import io.netty.util.concurrent.FailedFuture;
+import io.vertx.core.Future;
+import io.vertx.core.json.JsonObject;
+import org.apache.commons.pool2.impl.GenericObjectPool;
+import org.apache.commons.pool2.impl.GenericObjectPoolConfig;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.*;
+import java.nio.ByteBuffer;
+import java.time.*;
+import java.time.format.DateTimeFormatter;
+import java.util.*;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionException;
+import java.util.function.Supplier;
+import java.util.zip.*;
+
+import static org.mockito.Mockito.mock;
+
+/**
+ * @author gaurav.ashok
+ */
+public class MockAggregatedProfile {
```

Comment:
Move out the test in this to AggregatedProfileSerdeTest. This file could have generic methods only to generate mock window. Can this be moved to aggregator/storage module. Not really a e2e test